pkg/client/rootd/dns/server.go

```diff
@@ -633,10 +636,11 @@ func (s *Server) resolveQuery(q *dns.Question, dv *cacheEntry) (dnsproxy.RRs, in
 }
 
 // Run starts the DNS server(s) and waits for them to end.
-func (s *Server) Run(c context.Context, initDone chan<- struct{}, listeners []net.PacketConn, fallbackPool FallbackPool, resolve Resolver) error {
+func (s *Server) Run(c context.Context, initDone chan<- struct{}, listeners []net.PacketConn, fallbackPool FallbackPool, resolve Resolver, proxyCluster bool) error {
 	s.ctx = c
 	s.fallbackPool = fallbackPool
 	s.resolve = resolve
+	s.proxyCluster = proxyCluster
```

Comment:
Your editor and grep are correct.I added it there and then redesigned some stuff that rendered it unused. I left it in place anyway because I think we have more work to do in this area. Linux will simply select the `overridingResolver` when the cluster isn't proxied, but on Windows I think we might come up with a different solution.